kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py

```diff
@@ -51,111 +54,126 @@ def sample_code_ray_orchestrator(
     p2_pipeline_runtime_actor_options: str = '{"num_cpus": 0.8}',
     p2_pipeline_data_max_files: int = -1,
     p2_pipeline_data_num_samples: int = -1,
-    # Exact dedup step parameters
-    p3_name: str = "ededup",
+    # code to parquet step parameters
+    p3_name: str = "code2parquet",
     p3_skip: bool = False,
-    p3_ededup_doc_column: str = "contents",
-    p3_ededup_hash_cpu: float = 0.5,
-    # data sampling
-    p3_ededup_n_samples: int = 10,
+    # code to parquet parameters
+    p3_code_to_parquet_supported_langs_file: str = "test/code_2_parquet/languages/lang_extensions.json",
+    p3_code_to_parquet_detect_programming_lang: bool = True,
+    p3_code_to_parquet_domain: str = "code",
+    ip3_ngest_to_parquet_snapshot: str = "github",
+    p3_code_to_parquet_s3_access_secret: str = "s3-secret",
     # overriding parameters
     p3_overriding_params: str = '{"ray_worker_options": {"image": "'
+    + code_to_parquet_image
```

Comment:
strange indentation/new line separations, can you check it please

Reply:
It is a result of the "pre-commit" run.